src/components/helpers/allfilms.js

```diff
@@ -0,0 +1,13 @@
+exports.allfilms = async () => {
+	  const data = await fetch("https://sonal-hasura.herokuapp.com/v1/graphql", {
+            "method": "POST",
+            "headers": {
+            "Content-Type": "application/json"
+            },
+                "body": "{\"query\":\"{\\n tnff_films {\\n                id\\n                title\\n                tnff_year\\n                location\\n                category\\n                release_date\\n                duration_mins\\n                language\\n                director\\n                producer\\n                synopsis\\n                director_bio\\n                film_img_url\\n                director_img_url\\n                other_info\\n            }\\n}\"}"
```

Comment:
also this, is there a way to make this a prettier formatting, it's hard to read the query.

Reply:
cannot separate these onto different lines